src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              class="nav_link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
```

Comment:
```suggestion
              class="nav__link"
```

Please pay attention to class naming - there should be 2 (two) underlines instead of one (in BEM module you'll discover, why)